lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +53,48 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def check_approvals
+        has_comments_or_approvals = (has_comments? or approvals.any?)
+
+        case self.class.minimum_approvals
+        when "0"
+          true
+        when "", nil
+          # Approvals from every commenter
+          has_comments_or_approvals && reviewers_pending_response.empty?
+        else
+          approvals.size >= self.class.minimum_approvals.to_i and !last_comment.match(self.class.approval_regex).nil?
+        end
+      end
+
+      # Standard Rejection Messages for Error with Reflow Deliver
```

Comment:
Remove comment please.  The tests should cover this.  I don't see a test for this method.  I do realize we are testing it by proxy with the deliver command specs, but I think it should have it's own unit test.